src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/components/_misc.scss

```diff
@@ -16,14 +16,14 @@ a {
   display: none;
 }
 
-// DEFINE THE COMPANY LOGO HERE
 .logo {
-  @include svg-with-png-fallback(logo_sumocoders, #e84914 no-repeat center center, $dimensions: false);
+  background-color: #e84914;
```

Comment:
Als ik me niet vergis is dit de achtergrondkleur achter het logo, niet logischer om hier een variabele van te maken?